nvtabular/columns/selector.py

```diff
@@ -84,14 +86,14 @@ def _nested_check(self, nests=0):
         for col_sel0 in self.subgroups:
             col_sel0._nested_check(nests=nests + 1)
 
-    def __getitem__(self, index):
-        return self._names[index]
+    # def __getitem__(self, index):
+    #     return self._names[index]
 
-    def __len__(self):
-        return len(self.names)
+    # def __len__(self):
+    #     return len(self.names)
 
-    def __iter__(self):
-        return iter(self.names)
+    # def __iter__(self):
+    #     return iter(self.names)
```

Comment:
Let's go ahead and remove this instead of commenting it out. If we need to retrieve it later for some reason, we can always find it in the commit history